Clinical-Genomics/cg
# Skips second scout upload if analysis upload started twice, resolves #279

This PR fixes the problem that arises when you try to upload a case twice simultaneously

Contact: @patrikgrenfeldt @emmser 

How to test:
0. install on stage rasta
0.1. run cg/scripts/add-upload_started_at-to-analysis.sql on the cg-stage database
1. us
1. find an analysis that can be uploaded 
1. cg upload -f 'family_id' &
1. cg upload -f 'family_id'

Expected outcome:

- [x] The first cg upload invokation should start an upload the second should instead report that it has already started to upload `analysis upload already started`
![image](https://user-images.githubusercontent.com/6697636/68572682-5bb53900-0466-11ea-8b6c-92bb655c24ef.png)

- [x] if the first upload started more than 24h ago, throw an exception
![image](https://user-images.githubusercontent.com/6697636/68572801-a040d480-0466-11ea-86ed-9d82af15265a.png)

- [x] upload should be restart-able with --restart  
![image](https://user-images.githubusercontent.com/6697636/68572893-d67e5400-0466-11ea-9459-ae3b458c13f9.png)


This is a minor **version bump** because it expands the interface in a backwards compatible manner
